src/jio-footer.ts

```diff
@@ -69,7 +69,9 @@ export class Footer extends LitElement {
                <jio-report-a-problem sourcePath=${this.sourcePath} githubRepo=${this.githubRepo} .githubBranch=${ifDefined(this.githubBranch)}></jio-report-a-problem>
             </p>
             <div class="license-box">
-              ${licenseHtmls[this.license] || html``}
+            <b>
+            ${licenseHtmls[this.license] || html``}
```

Comment:
why set font weight and wrap it in <b>
Can you also fix indenting?

Reply:
> why set font weight and wrap it in

Do you mean use css to bold the text ?
i'll fix the indenting

Reply:
![image](https://user-images.githubusercontent.com/110087/212566428-2807c063-cfd3-49eb-bc0e-f5628b600c71.png)
i'm saying you already added font-weight bold, why are you wrapping it in b as well?

I'd personally prefer css over html styling tags.